pom.xml

```diff
@@ -0,0 +1,366 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>io.cdap.plugin</groupId>
+  <artifactId>github-plugin</artifactId>
+  <name>GitHub plugin</name>
+  <version>1.0.0-SNAPSHOT</version>
+  <description>A collection of gitHub connectors and plugins</description>
+  <url>https://github.com/data-integrations/github</url>
+
+  <licenses>
+    <license>
+      <name>The Apache Software License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <distribution>repo</distribution>
+      <comments>A business-friendly OSS license</comments>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <name>CDAP</name>
+      <email>cdap-dev@googlegroups.com</email>
+      <organization>CDAP</organization>
+      <organizationUrl>http://www.cdap.io</organizationUrl>
+    </developer>
+  </developers>
+
+  <repositories>
+    <repository>
+      <id>sonatype</id>
+      <url>https://oss.sonatype.org/content/groups/public</url>
+    </repository>
+    <repository>
+      <id>sonatype-snapshots</id>
+      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+    </repository>
+    <repository>
+      <id>eclipse</id>
+      <url>https://repo.eclipse.org/content/groups/releases</url>
+    </repository>
+  </repositories>
+
+  <issueManagement>
+    <url>https://issues.cask.co/browse/CDAP</url>
+  </issueManagement>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <!-- properties for script build step that creates the config files for the artifacts -->
+    <widgets.dir>widgets</widgets.dir>
+    <docs.dir>docs</docs.dir>
+    <!-- this is here because project.basedir evaluates to null in the script build step -->
+    <main.basedir>${project.basedir}</main.basedir>
+    <cdap.version>6.1.0-SNAPSHOT</cdap.version>
+    <hadoop.version>2.3.0</hadoop.version>
+    <hydrator.version>2.3.0-SNAPSHOT</hydrator.version>
+    <junit.version>4.11</junit.version>
+    <mockito.version>1.10.19</mockito.version>
+    <spark.version>2.1.3</spark.version>
+    <assertj.version>3.11.1</assertj.version>
+    <random-beans.version>3.9.0</random-beans.version>
+    <google-http-client-gson.version>1.32.1</google-http-client-gson.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.cdap.cdap</groupId>
+      <artifactId>cdap-etl-api</artifactId>
+      <version>${cdap.version}</version>
```

Comment:
scope should be provided
https://github.com/data-integrations/google-cloud/blob/develop/pom.xml#L148

Reply:
Done